src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -160,6 +183,13 @@ const MenuContent = ({ programName }: { programName: string }) => {
 		},
 	});
 
+	const { isRDPCAdmin, isDCCAdmin, isProgramAdmin, isDataSubmitter, isCollaborator } = userRoles;
+
+	const userCanSubmitData = isDataSubmitter || isRDPCAdmin || isDCCAdmin || isProgramAdmin;
+	const userCanViewData =
+		isCollaborator || isDataSubmitter || isRDPCAdmin || isDCCAdmin || isProgramAdmin;
+	const userCanManageProgram = isDataSubmitter || isRDPCAdmin || isDCCAdmin || isProgramAdmin;
+
```

Comment:
What are your thoughts on exposing more specific functionality from within `useUserRoles`?
if we're just checking can user view data... can we just check for that specific thing? vs checking if a user is part of any role that has that permission. Does that make sense? These role booleans are just built from ego token checks. So here we're checking a bunch of supersets instead of just checking for the permission.

Reply:
We could check `${PROGRAM}.READ` / `${PROGRAM}.WRITE` permissions, which is how I approached this branch originally. See:
https://github.com/icgc-argo/rdpc-ui/pull/114/commits/d3b5622f7f958385b35e58165390ff5e5fbc49b3
(original code that I removed)

These booleans came from matching useUserRoles to the outlines of which roles should see which page in ticket #979:
https://app.zenhub.com/workspaces/icgc-argo-platform-dk-production-board-5e542d38415f5034e9fed89d/issues/gh/icgc-argo/roadmap/979